mappgene/subscripts/ivar.py

```diff
@@ -87,6 +91,7 @@ def run_ivar(params):
     output_tsv = join(output_dir, f'{subject}.ivar.tsv')
     output_fa = join(output_dir, f'{subject}.ivar.consensus')
     run(f'bwa index {fasta}', params)
+    run(f'fastp -i {read1} -I {read2} -o {r1_fastp} -O {r2_fastp} -D --trim_front1 {trim_front_tail} --trim_tail1 {trim_front_tail} -l 25 -w {thread_usage} -h {Subject_dir}'.fastp.html' 2>dev/null}', params) 
     run(f'bwa mem -t 8 {fasta} {read1} {read2} | samtools sort -o {bam}', params)
```

Comment:
We have to update this line to make sure that `bwa mem` gets the filtered input from `fastp` and not the "raw" fastq files as it does now.